wordpress-mobile/WordPress-Android
# Hotfix crash when loading vector placeholders on api < 21

Fixes #8155 

Fixes crash when loading a vector placeholder on API < 21. 

To test - test on api < 21
1. Go to Themes
2. Make sure the app doesn't crash
